localisation/00_wc_ruler_titles_castle.csv

```diff
@@ -164,3 +164,6 @@ duchy_satyr_group;Clan;Clan;Clan;;;;;;;;;;;x
 duchy_of_satyr_group;Tribe of;Tribu de;Stamm von;;;;;;;;;;;x
 county_of_satyr_group;Chiefdom of;Chefferie de;Stamm von;;;;;;;;;;;x
 county_satyr_group;Chiefdom;Chefferie;Stamm;;;;;;;;;;;x
+# forsaken;;;;;;;;;;;;;;x
+king_forsaken;Banshee King;;;;;;;;;;;;;x
+king_female_forsaken;Banshee Queen;;;;;;;;;;;;;
```

Comment:
Since "Banshee" is specific undead type, how about something... less specific?
"Dark Lady/Lord", for example?

Reply:
agree, we could give sylvanas banshee queen nickname

Reply:
I would use Dark Lady for duke tier.
Forsaken Queen or just Queen for king tier.
https://wow.gamepedia.com/Sylvanas_Windrunner

Reply:
Used Dark Lord/Lady, purged Banshee King/Queen in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1125/commits/241d36c5c7f6fab9a21d03f6da5229f0ea79e208.